components/payment-options.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+
+var PayPalButton = React.createClass({
+  mixins: [require('react-intl').IntlMixin],
+  propTypes: {
+    onClick: React.PropTypes.func,
+    submitting: React.PropTypes.bool,
+    validate: React.PropTypes.array,
+    submit: React.PropTypes.array.isRequired
+  },
+  onChange: function() {
+    this.props.onClick = this.props.onClick || function() {};
```

Comment:
onClick is used in sequential form to toggle the credit card details on and off.